doc/ref_model/chapters/chapter01.md

```diff
@@ -12,6 +12,18 @@
 ## 1.1 Overview
 The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by Cloud iNfrastructure Telco Taskforce (CNTT).  For more information about CNTT, its activities and deliverables, see [Cloud iNfrastructure Telco Taskforce](../../common/chapter00.md#2.0).
 
+**Problem Statement:**
+Based on community consultations, including telco operators, technology suppliers, and software developers, there is a realisation that there are significant technical, operational and business challenges to the development and deployment of VNF/CNF network applications related to the lack of a common cloud infrastructure platform. These include but are not limited to the following:
+
+    - Higher development costs due to the need to develop virtualised/containerised network applications on multiple custom platforms for each operator
+    - Increased complexities due to the need to maintain multiple versions of applications to support each custom environment
```

Comment:
Need consistency of end full-stop. Not used in the first two bullets, is used in the last three.